test/run_test.py

```diff
@@ -340,6 +349,82 @@ def print_to_stderr(message):
     print(message, file=sys.stderr)
 
 
+def get_test_time_reports_from_S3():
+    commit_date_str = subprocess.check_output(
+        ['git', 'show', '-s', '--format=%ci', 'HEAD'],
+        encoding="ascii").splitlines()[0]
+    commit_date = datetime.strptime(commit_date_str, '%Y-%m-%d %H:%M:%S %z')
```

Comment:
Let computers speak computer dates with each other ('%ci' will return a unix-timestamp), see:
```
% git show -s --format=%ct HEAD                                                       
1614093721
% python3 -c "from datetime import datetime;print(datetime.fromtimestamp(1614093721))"
2021-02-23 07:22:01
```
```suggestion
    commit_date_ts = subprocess.check_output(
        ['git', 'show', '-s', '--format=%ct', 'HEAD'],
        encoding="ascii").strip()
    commit_date = datetime.fromtimestamp(int(commit_date_ts))
```